src/index.ts

```diff
@@ -1,45 +1,157 @@
 function _(selector: string): any {
-	/**
-	 * innerHTML() {
-	 * }
-	 *
-	 * show() {
-	 * }
-	 *
-	 * hidden() {
-	 * }
-	 *
-	 * addEvent() {
-	 * }
-	 */
+  function innerHTML(template?: string) {
+    const el = document.getElementById(selector) as HTMLElement;
+    if (template) {
+      el.innerHTML = template;
+      return;
+    }
+
+    return el.innerHTML;
+  }
+
+  function show() {
+    const el = document.getElementById(selector) as HTMLElement;
+    el.style.display = '';
+  }
+
+  function hidden() {
+    const el = document.getElementById(selector) as HTMLElement;
+    el.style.display = 'none';
+  }
+
+  function addEvent(type: keyof HTMLElementEventMap, listener: EventListenerOrEventListenerObject) {
+    const el = document.getElementById(selector) as HTMLElement;
+    el.addEventListener(type, listener);
+  }
+
+  return {
+    innerHTML,
+    show,
+    hidden,
+    addEvent,
+  };
 }
 
 module _ {
-	export function fetch() {
-		return {};
-	}
+  export function fetch(path: RequestInfo | URL, config?: RequestInit): Promise<Response> {
+    return window.fetch(path, config).then((response) => {
+      if (!response.ok) {
+        throw new Error(response.statusText);
+      }
+      return response.json();
+    });
+  }
+
+  type Nullable<Type> = Type extends null ? Type : never;
+  export function isNull<Type>(args: Type): args is Nullable<Type> {
+    if (args === null) {
+      return true;
+    }
+
+    return false;
+  }
+
+  type Nillable<Type> = Type extends null | undefined ? Type : never;
+  export function isNil<Type>(args: Type): args is Nillable<Type> {
+    if (args === null || args === undefined) {
+      return true;
+    }
+
+    return false;
+  }
+
+  export function isNumber(args: unknown): args is number {
+    return typeof args === 'number';
+  }
+
+  export function isFunction(args: unknown): args is Function {
+    return typeof args === 'function';
+  }
+
+  export function shuffle(args: any[]): any[] {
```

Comment:
```suggestion
  export function shuffle<T>(args: T[]): T[] {
```
제네릭으로 타입 추론이 가능할것같아요!

Reply:
좋습니다!~!